### ISSUE TITLE
Support for eth.getStorageAt with eth-tester

### ISSUE THREAD

**Issue by carver** [ethereum/web3.py#1490]
### What was wrong?

`eth_getStorageAt` is unimplemented in eth-tester, and also in the web3.py adapter.

### How can it be fixed?

Add the endpoint to eth-tester, then add it to the web3 middleware. Should be straightforward.

This is obviously not the "right" way to do it, but for reference on where the info is, I hacked around the problem with:

```py
w3.provider.ethereum_tester.backend.chain.get_vm().state.get_storage(address, slot)
```

**Comment by MatthiasLohr** [comment]
Would need that, too. However, I even failed to try your dirty hack so most probably the wrong person for trying to implement in a good way :/

**Comment by fselmo** [comment]
closed by #3011
